public/app/features/dashboard-scene/serialization/layoutSerializers/utils.ts

```diff
@@ -152,3 +153,36 @@ export function getLayout(sceneState: DashboardLayoutManager): DashboardV2Spec['
   }
   return registryItem.serializer.serialize(sceneState);
 }
+
+// Functions to manage the lookup table in dashboard scene that will hold element_identifer : panel_id
+export function getElementIdentifierForVizPanel(vizPanel: VizPanel): string {
```

Comment:
Probably this could be exposed at `dashboardSceneGraph.getElementIdForVizPanel()`, so this can be generally used. For a given element, we find the DashboardScene, and we get it using the serializer.

Reply:
done